uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/instances/leave-request.instance.js

```diff
@@ -52,6 +35,32 @@ define([
           }.bind(this));
       }
 
+      /**
+       * Checks if the balance change of the leave request need to be recalculated
+       *
+       * @return {Promise} resolves with {Boolean}
+       */
+      function checkIfBalanceChangeNeedsForceRecalculation () {
```

Comment:
`checkIfBalanceChangeNeedsRecalculation` (no need for the word Force)

Reply:
Because sometimes you won't force the user to recalculate

Reply:
Done.